.github/workflows/gh-pages.yml

```
@@ -204,31 +204,42 @@
-          path: /Users/runner/Library/Caches/pypoetry/virtualenvs
-          key: python-${{ runner.os }}-poetry-${{ hashFiles('**/poetry.lock') }}
-          restore-keys: python-${{ runner.os }}-poetry-
-
       - name: Install Dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
         working-directory: fonts
         run: |
           poetry install --no-root
 
+      - name: Make Directory
+        if: steps.cache.outputs.cache-hit != 'true'
```

actionlint (1.7.4)

property "cache" is not defined in object type {}

(expression)